scripts/profile-time.py

```diff
@@ -0,0 +1,115 @@
+import sys
```

Comment:
may help to include some examples of how you run this here or in the help/usage below

Reply:
7a4aee592be171c7ba7fde03623a1596db831b9e